tests/rkt_tests.go

```diff
@@ -142,6 +142,39 @@ func waitOrFail(t *testing.T, child *gexpect.ExpectSubprocess, expectedStatus in
 	}
 }
 
+// waitPodReady waits for the pod supervisor to get ready, busy-looping `loops` times
+// while waiting for it. It returns the pod UUID or an error on failure.
+func waitPodReady(ctx *testutils.RktRunCtx, uuidFile string, loops int) (string, error) {
```

Comment:
nit: I think loops -> time.Duration would give a nicer method signature.

The busy-loop could still sleep in .5s chunks until it reaches that duration.